Tests/TuistSupportTests/Utils/GlobTests.swift

```diff
@@ -0,0 +1,360 @@
+import Foundation
+import XCTest
+@testable import TuistSupport
+
+//
+//  Created by Eric Firestone on 3/22/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//  Released under the Apache v2 License.
+//
+//  Adapted from https://gist.github.com/blakemerryman/76312e1cbf8aec248167
+
+final class GlobTests: XCTestCase {
+    let tmpFiles = ["foo", "bar", "baz", "dir1/file1.ext", "dir1/dir2/dir3/file2.ext", "dir1/**(_:_:)/file3.ext"]
+    var tmpDir: URL!
+
+    override func setUp() {
+        super.setUp()
+
+        let uuid = ProcessInfo.processInfo.globallyUniqueString
```

Comment:
I don't follow why we need to access a UUID through `ProcessInfo`? If we used `fileHandler.temporaryDirectory()` method, you wouldn't need to generate your own UUID at all

Reply:
Yeah I don't get it too lol. Just removed it by using `FileHander` instead.